src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && <p className="Person__age">I am 37</p>}
+
+      {!isMarried && <p className="Person__partner">I am not married</p>}
+
+      {sex === 'm' ? (
+        <p className="Person__partner">{`${partnerName} is my wife`}</p>
+      ) : (
+        <p className="Person__partner">{`${partnerName} is my husband`}</p>
```

Comment:
You can create a variable for `partnerStatus`
`const partnerStatus = sex === 'm' ? 'wife' : 'husband';`
and then :

`<p className="Person__partner">{${partnerName} is my ${partnerStatus}}</p>`